### ISSUE TITLE
Do not fetch exchange rates when there is no server connection.

### ISSUE THREAD

**Issue by rmisio** [OpenBazaar/openbazaar-desktop#263]
The exchange rate syncer fetches exchange rates on an interval. But, if we lose the connection with the server, the interval should be paused.

**Comment by rmisio** [comment]
This depends on code from #262.

**Comment by jjeffryes** [comment]
This is similar to #367, both could be done together.

**Comment by hoffmabc** [comment]
OK #260 262 was closed and merged. Where we at with this one?

**Comment by jjeffryes** [comment]
I'm pretty sure no server calls are made when the server is not connected. @rmisio can you confirm whether this can be closed?

**Comment by rmisio** [comment]
No, this still needs to be done. I think this could be considered low priority because the calls only happen every 15 minutes, are local and if there is no connection, they'll just relatively gracefully fail.
